docs/hugo/content/introduction/metrics.md

```diff
@@ -0,0 +1,54 @@
+# ASOv2 Metrics
+
+Prometheus metrics are exposed for ASOv2 which can be helpful in diagnosability and tracing.
+The metrics exposed fall into two groups: Azure based metrics and reconciler metrics (from `controller-runtime`).
+
+## Disabling the metrics
+
+By default, metrics scraping is turned on for ASOv2. It can be turned off using the Helm chart 
+`--set metrics.enable` or by omitting `--metrics-addr` flag from the Deployment yaml.
```

Comment:
Why does `--set metrics.enable` turn OFF metrics scraping?

Reply:
Oh, you cover the details below. Perhaps reword this paragraph to make it read more like an introduction.

Reply:
Makes sense.. I've made it more clear in the recent commit.